lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/dcmawCri/ContractTest.java

```diff
@@ -0,0 +1,2788 @@
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String FAILED_DVA_VC_NO_CI_SIGNATURE =
+            "Vfn6YWxKlwwdUeau-o_2Qx67ni2KMp6JPseqShrP9lcQskNaoBMpxFeae-Q5VAWlVcZlry9D9cU7tsrr6r2WYA";
+
+    // 2099-01-01 00:00:00 is 4070908800 in epoch seconds
+    // From DCMAW-1559-AC2
+    private static final String VALID_DVA_VC_BODY =
+            """
+            {
+              "sub": "test-subject",
+              "iss": "dummyDcmawComponentId",
+              "nbf": 4070908800,
+              "exp": 4070909400,
+              "vc": {
+                "@context": [
+                  "https://www.w3.org/2018/credentials/v1",
+                  "https://vocab.account.gov.uk/contexts/identity-v1.jsonld"
+                ],
+                "type": [
+                  "VerifiableCredential",
+                  "IdentityCheckCredential"
+                ],
+                "credentialSubject": {
+                  "name": [
+                    {
+                      "nameParts": [
+                        {
+                          "value": "OLGA",
+                          "type": "GivenName"
+                        },
+                        {
+                          "value": "A",
+                          "type": "GivenName"
+                        },
+                        {
+                          "value": "KULYK",
+                          "type": "FamilyName"
+                        }
+                      ]
+                    }
+                  ],
+                  "birthDate": [
+                    {
+                      "value": "1980-09-27"
+                    }
+                  ],
+                  "deviceId": [
+                    {
+                      "value": "a3017511-b639-46ff-ab73-66e5ab0193c9"
+                    }
+                  ],
+                  "address": [
+                    {
+                      "uprn": null,
+                      "organisationName": null,
+                      "subBuildingName": null,
+                      "buildingNumber": null,
+                      "buildingName": null,
+                      "dependentStreetName": null,
+                      "streetName": null,
+                      "doubleDependentAddressLocality": null,
+                      "dependentAddressLocality": null,
+                      "addressLocality": null,
+                      "postalCode": null,
+                      "addressCountry": null
+                    }
+                  ],
+                  "drivingPermit": [
+                    {
+                      "expiryDate": "2028-02-20",
+                      "issueDate": "2022-05-29",
+                      "issueNumber": null,
+                      "issuedBy": "DVA",
+                      "personalNumber": "5823131861",
+                      "fullAddress": null
```

Comment:
Will we have null fields here? Or will they just be missing from the VC body?

Not sure if this will make a difference or not in our PACT file, but I remember doing the driving license variations for F2F Message Queue and I believe the fields will just be missing entirely, rather than having a null value

Reply:
This is taken from a DCMAW unit test so I guess it will have an actual null value